### ISSUE TITLE
f key not working from time to time

### ISSUE THREAD

**Issue by Mo2000** [akhodakivskiy/VimFx#520]
I am unsure on what is causing this. The f would just stop working and cause the all other keys to be disabled. I would have to press Esc to get the other keys working.

I don't remember changing anything since the last this was working (last week).

This happens on all websites, and not just on ones that contain SVG.

I'm unsure as to where to get an error log for this addon (if one exists). I'd be willing to provide more info if need be.

EDIT: Just to be a bit more precise. The f would not start working again until I reboot the computer. Restarting the browser does not solve it either.

**Comment by lydell** [comment]
You can find the error log in the Browser Console. It can be opened from Web Developer section in the menu, or by pressjng `<c-J>`.

**Comment by Mo2000** [comment]
I just tried opening the console, reloading the page, and hitting f, but noting new shows up in the console that pertains to hitting the f key.

**Comment by lydell** [comment]
Will be fixed by #378.

**Comment by Mo2000** [comment]
Thank you, but just to update. I was able to resolve this problem by resetting Firefox completely. I'm not sure what the exact reason was, but I THINK another add-on that I had installed had caused this somehow. I have no proof, but I had the add-on I had installed was https://addons.mozilla.org/en-US/firefox/addon/nimbus-screenshot/?src=search and I think that it may have had something to do with the issue. I'm stating this if anyone wants to look into it.

Thanks for the updates!
